src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -120,9 +143,9 @@ public void ClosePin(int pinNumber)
         /// </summary>
         /// <param name="pinNumber">The pin number in the controller's numbering scheme.</param>
         /// <param name="mode">The mode to be set.</param>
-        public void SetPinMode(int pinNumber, PinMode mode)
+        public virtual void SetPinMode(int pinNumber, PinMode mode)
```

Comment:
if ClosePin takes numbering scheme why doesn't this? (similar comment elsewhere)

Reply:
We could add these overloads, yes. This would allow specifying, for a particular call, that you use another than the default numbering scheme. But I'd rather do that on a separate PR. For the close, it is required for different reasons (see above).